sdk/servicebus/azure-servicebus/README.md

```diff
@@ -97,6 +93,12 @@ To interact with these resources, one should be familiar with the following SDK
 
 * [ServiceBusMessage][message_reference]: When sending, this is the type you will construct to contain your payload.  When receiving, this is where you will access the payload.
 
+### Thread safety
+
+We do not guarantee that the clients are thread-safe. We do not recommend reusing client instances across threads. It is up to the running application to use the clients in a thread-safe manner.
+
+The data model types, such as `EventDataBatch` are not thread-safe. They should not be shared across threads nor used concurrently with client methods.
```

Comment:
I doubt anyone is using an `EventDataBatch` in Service Bus ;)
I don't think there's even an equivalent in SB? So probably all the models are fine?

Reply:
Ah I copy/pasted, will remove